src/main/java/org/honton/chas/exists/RemoteExistsMojo.java

```diff
@@ -230,8 +230,16 @@ boolean resourceExists(String resourceName) throws Exception {
       return wagon.resourceExists(resourceName);
     }
 
-    void get(String resourceName, OutputStream outputStream) throws Exception {
-      ((StreamingWagon) wagon).getToStream(resourceName, outputStream);
+    String getContent(String resourceName) throws Exception {
+      if (wagon instanceof StreamingWagon) {
+        ByteArrayOutputStream baos = new ByteArrayOutputStream();
+        ((StreamingWagon) wagon).getToStream(resourceName, baos);
+        return new String(baos.toByteArray(), StandardCharsets.ISO_8859_1);
```

Comment:
extract byte[] in each leg of the if/else, and put common code of creating the String outside of the if/else